test/cluster.sh

```diff
@@ -78,32 +92,28 @@ function install_knative_serving() {
 
 # Uninstalls Knative Serving from the current cluster.
 function uninstall_knative_serving() {
-  create_manifests
+  if [[ -z "${INSTALL_RELEASE_YAML}" ]]; then
+    echo "install_knative_serving() was not called, nothing to uninstall"
+    return 0
+  fi
+  echo ">> Uninstalling Knative serving"
+  echo "Istio YAML: ${INSTALL_ISTIO_YAML}"
+  echo "Knative YAML: ${INSTALL_RELEASE_YAML}"
   echo ">> Removing test resources (test/config/)"
   ko delete --ignore-not-found=true -f test/config/ || return 1
 
   echo ">> Bringing down Serving"
-  # TODO(#2122): Use RELEASE_YAML once we have monitoring e2e.
-  if [[ -z "${RELEASE_YAML_OVERRIDE}" ]]; then
-    ko delete --ignore-not-found=true -f "${RELEASE_NO_MON_YAML}" || return 1
-  else
-    ko delete --ignore-not-found=true -f "${RELEASE_YAML_OVERRIDE}" || return 1
-  fi
+  ko delete --ignore-not-found=true -f "${INSTALL_RELEASE_YAML}" || return 1
 
   echo ">> Bringing down Istio"
-  kubectl delete --ignore-not-found=true -f ${ISTIO_YAML} || return 1
+  kubectl delete --ignore-not-found=true -f "${INSTALL_ISTIO_YAML}" || return 1
```

Comment:
Don't we also need to delete `$INSTALL_ISTIO_CRD_YAML`?

Reply:
`istio.yaml` also contains the CRDs, so they'll be deleted here. The separate CRD yaml is to avoid a race condition when installing istio, and should be dropped once this is fixed in istio (there's an issue somewhere with details).